packages/authentication/test/unit/fixtures/mock-strategy.ts

```diff
@@ -31,13 +35,16 @@ export class MockStrategy implements Strategy {
    * pass req.query.testState = 'fail' to mock failed authorization
    * pass req.query.testState = 'error' to mock unexpected error
    */
-  async verify(req: http.ServerRequest) {
+  async verify(this: StrategyCreated<this> & this, request: Express.Request) {
+    // A workaround for buggy typings provided by @types/passport
```

Comment:
I think we should be good citizens and push a fix upstream for the typings rather than hack our own stuff to work around the damage.